src/components/Button/index.js

```diff
@@ -34,11 +34,17 @@ export default class Button extends PureComponent {
         background: ${({ primary }) =>
           primary ? theme.color.primaryLight : theme.color.greyLightest};
       }
+
+      :disabled {
```

Comment:
Did you test this in the browser? I'm not sure that'll work... I think it has to be like this (similar to scss):

```scss
&:disabled {
  // ... styles
}
```

Reply:
Yes - I developed this against Storybook in Chrome Canary and this works.

According to the Styled Components docs:

> Since we generate an actual stylesheet and not inline styles, whatever works in CSS works in styled-components!
  - via https://www.styled-components.com/docs/api#supported-css

Since `:disabled` is a legal [pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/:disabled), it works. I admit, it felt weird, but I pulled down the styled-components repo myself and added a test for `disabled` and got the expected result. Basically, what they do is turn a ruleset like this:

```js
div`
  color: blue;
  font-size: 1em;

  &:hover {
    color: orange;
  }

  :disabled {
    opacity: 0.5
  }
`
```

into:

```css
div {
  color: blue;
  font-size: 1em;
}

div:hover {
  color: orange;
}

div:disabled {
  opacity: 0.5
}
```

This happens [here](https://github.com/styled-components/styled-components/blob/master/src/utils/flatten.js) if you're interested.

Now, having said all that, if you would prefer I use SASS-style syntax, I would be happy to change it. This is admittedly not clearly documented in the styled-components ecosystem so I am going to consider making a PR over there as well.

Reply:
TIL! That's pretty sweet